torch/testing/_internal/common_methods_invocations.py

```diff
@@ -1047,53 +1047,53 @@ def sample_inputs_fliplr_flipud(op_info, device, dtype, requires_grad):
 # Foreach pointwise ops
 foreach_pointwise_op_db: List[OpInfo] = [
     ForeachFuncInfo('_foreach_addcmul',
-                         method=torch._foreach_addcmul,
-                         inplace=torch._foreach_addcmul_,
-                         ref=torch.addcmul,
-                         dtypes=all_types(),
-                         dtypesIfCPU=all_types(),
-                         dtypesIfCUDA=all_types()),
+                    method=torch._foreach_addcmul,
+                    inplace=torch._foreach_addcmul_,
+                    ref=torch.addcmul,
+                    dtypes=all_types(),
```

Comment:
@ngimel had an interesting idea that these could acquire their dtypes from the reference operation's OpInfo, at least when that OpInfo exists. For example, the OpInfo for asin can be acquired by matching reference and the .op attribute of the OpInfo.